api/src/main/java/org/openmrs/api/context/UserContext.java

```diff
@@ -191,6 +202,7 @@ public boolean isAuthenticated() {
 	 */
 	public void logout() {
 		log.debug("setting user to null on logout");
+    notifyUserSessionListener(user, Event.LOGOUT, Status.SUCCESS);
```

Comment:
Indentation is missed here. Can you please recheck all the changes for the indentation. 
It caused default by the IDE also

Reply:
o, i forgot, my eclipse was set on a 2 space indention in contrast to our tab indention. good catch @suthagar23